src/lib.rs

```diff
@@ -112,6 +124,13 @@ impl HTTPMetricsLayerBuilder {
         HTTPMetricsLayerState {
             server_request_duration: meter
                 .f64_histogram(Cow::from(HTTP_SERVER_DURATION_METRIC))
+                .with_unit(Cow::from(HTTP_SERVER_DURATION_UNIT))
+                .with_boundaries(HTTP_SERVER_DURATION_BOUNDARIES.to_vec())
```

Comment:
This is not needed - the histogram boundaries get set to the correct default values here: https://github.com/open-telemetry/opentelemetry-rust/blob/main/opentelemetry-sdk/src/metrics/pipeline.rs#L453 

As I was digging into that I also noticed that the SDK uses values for bucket boundaries that are 100x what is in the spec - I have not found exactly where that comes into play but it seems like an optimization they chose and best to let the sdk handle this internally since it is designed to give us spec compliance by default.

```rust
InstrumentKind::Histogram => Aggregation::ExplicitBucketHistogram {
            boundaries: vec![
                0.0, 5.0, 10.0, 25.0, 50.0, 75.0, 100.0, 250.0, 500.0, 750.0, 1000.0, 2500.0,
                5000.0, 7500.0, 10000.0,
            ],
            record_min_max: true,
        },
```

Reply:
Happy to remove but [this](https://opentelemetry.io/docs/specs/semconv/http/http-metrics/#metric-httpserverrequestduration) reads to me that this particular `http.server.request.duration` metric should explicitly use these much smaller boundaries, independant of what the  default values are for the Histogram instrument.

Reply:
huh yeah you're right I had assumed these were percentile buckets base on the values chosen but they're literal value boundaries after looking into the code.

I chased down more discussion in the opentlemetry-go repo, and it says this was chosen to match Prometheus but still needs fixed for request durations https://github.com/open-telemetry/opentelemetry-specification/pull/2770/files